openxc-ios-framework/VehicleManager.swift

```diff
@@ -747,6 +778,49 @@ open class VehicleManager: NSObject, CBCentralManagerDelegate, CBPeripheralDeleg
     ]
     
     // decode the command type and build the command depending on the command
+
+    if cmd.command == .version || cmd.command == .device_id || cmd.command == .sd_mount_status || cmd.command == .platform {
```

Comment:
looks like some issue in merging conflicts. These were removed as part of last PR merged in framework